LEAF_Request_Portal/templates/reports/LEAF_Inbox.tpl

```diff
@@ -109,6 +109,20 @@
                     }
                 }
             },
+            'type': function(site) {
+                return {
+                    name: 'Type',
+                    indicatorID: 'type',
+                    editable: false,
+                    callback: function(data, blob) {
+                        let types = '';
```

Comment:
Something like this would be cleaner.  Wouldn't require trimming the string.

let typesArray = [];
for (let i in blob[data.recordID].categoryNames) {
    typesArray.push(blob[data.recordID].categoryNames[i]);
}
let types = typesArray.join(' | ');

Reply:
True. I'll update the other template where I copied this from too.

Reply:
`document.querySelector(`#${data.cellContainerID}`).innerHTML = blob[data.recordID].categoryNames.join(' | ');` This looks like it was all distilled down to a one liner.